### ISSUE TITLE
Upgrading to new MDBenchmark version breaks environment

### ISSUE THREAD

**Issue by mimischi** [bio-phys/MDBenchmark#127]
With our upgrade to `datreant>1.0`, we need to tell our users to uninstall `datreant.core` and `datreant.data` from their Python/Conda environments. Otherwise execution of `mdbenchmark` will not work.

I wrote a small workaround, where we use `console.error` to warn the user, that their current environment is incompatible with MDBenchmark v2.0 and they should delete those packages.

This is really ugly, but I guess there is no other way around this.. @kain88-de?

**Comment by kain88-de** [comment]
Yeah it’s a bit ugly. We recommend to update the main environment to
datreant>1 and keep another for an earlier version if you do not want to
update the treant directories. But there is no other way around to
uninstall datreant.data. The datreant.core package only exists on conda.
